docs/REST API/rest-api.md

```diff
@@ -1823,6 +1823,9 @@ None.
 - [Get host OS versions](#get-host-os-versions)
 - [Get hosts report in CSV](#get-hosts-report-in-csv)
 - [Get host's disk encryption key](#get-hosts-disk-encryption-key)
+- [Get host's scripts](#get-hosts-scripts)
+- [Get host's upcoming activities](#get-hosts-upcoming-activities)
+- [Get host's past activities](#get-hosts-upcoming-activities) 
```

Comment:
Heads-up, the link is to the `upcoming` section.

Reply:
✅